install/framework/nunit-framework.wxs

```diff
@@ -58,7 +58,7 @@
     
 	<Fragment>
 		<Directory Id="TARGETDIR" Name="SourceDir">
-			<Directory Id="ProgramFilesFolder">
+			<Directory Id="ProgramFiles64Folder">
```

Comment:
`ProgramFiles64Folder` goes to `Program Files`, `ProgramFilesFolder` goes to `Program Files (x86)`.

It confused me, same? :smile: